### ISSUE TITLE
Distortion and TCA profile for Nikon Coolpix P1000

### ISSUE THREAD

**Issue by Macchiato17** [lensfun/lensfun#1148]
I created the profile for this camera with respect to the complete focal coverage of 24-3000mm (KB equivalent) or 4,3-539mm (native) in 32 steps.
[Nikon_Coolpix_P1000_lensfun.zip](https://github.com/lensfun/lensfun/files/4512054/Nikon_Coolpix_P1000_lensfun.zip)

**Comment by Macchiato17** [comment]
After no action was taken on this issue (#1148) and I found some time in the meantime, I was trying to provide my changes directly via a pull request. The first one (#1169) got stuck after a successful build. Therefore I created a new one today (#1186) which also got stuck in the same status "continuous-integration/appveyor/pr Pending — Waiting for AppVeyor build to complete". 
So I'm lost at this point, may anyone kindly help me to solve this issue? Thanks in advance...
